tetris.py

```diff
@@ -300,13 +301,45 @@ def full_line_clear(self):
         for x_line in range(game_height):
             count = coordinates_counter[x_line]
             if count == game_width:
+                print(coordinates_counter)
+                print(y_coordinates)
+                self.flasher()
```

Comment:
When this code runs, you have already figured out which line needs to be flashed, and there's no need to do it again:

```suggestion
                self.flasher(x_line)
```